File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    const arr1Sum = arr1.reduce((acc, cur) => acc + cur, 0);
    const arr2Sum = arr2.reduce((acc, cur) => acc + cur, 0);

    if (arr1.length === arr2.length) {
        if (arr1Sum > arr2Sum) {
            return 1;
        } else if (arr2Sum > arr1Sum) {
            return -1;
        } else {
            return 0;
        }
    }

    return arr1.length > arr2.length ? 1 : -1;
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    arr1_sum = sum(arr1)
    arr2_sum = sum(arr2)

    if len(arr1) == len(arr2):
        if arr1_sum > arr2_sum:
            return 1
        elif arr2_sum > arr1_sum:
            return -1
        else:
            return 0

    return 1 if len(arr1) > len(arr2) else -1